docs/lineage/gravitino-spark-lineage.md

```diff
@@ -0,0 +1,109 @@
+
+| Dataset Type    | Dataset name                                         | Example                    | Since Version    |
+|-----------------|------------------------------------------------------|----------------------------|------------------|
+| Hive catalog    | `${GravitinoCatalogName}.${schemaName}.${tableName}` | `hive_catalog.db.student`  | 0.9.0-incubating |
+| Iceberg catalog | `${GravitinoCatalogName}.${schemaName}.${tableName}` | `iceberg_catalog.db.score` | 0.9.0-incubating |
+| Paimon catalog  | `${GravitinoCatalogName}.${schemaName}.${tableName}` | `paimon_catalog.db.detail` | 0.9.0-incubating |
+| JDBC catalog    | `${GravitinoCatalogName}.${schemaName}.${tableName}` | `jdbc_catalog.db.score`    | 0.9.0-incubating |
+
+For datasets not managed by Gravitino, the dataset name is as follows:
+
+| Dataset Type | Dataset name                                | Example                               | Since Version    |
+|--------------|---------------------------------------------|---------------------------------------|------------------|
+| Hive         | `spark_catalog.${schemaName}.${tableName}`  | `spark_catalog.db.table`              | 0.9.0-incubating |
+| Iceberg      | `${catalogName}.${schemaName}.${tableName}` | `iceberg_catalog.db.table`            | 0.9.0-incubating |
+| JDBC v2      | `${catalogName}.${schemaName}.${tableName}` | `jdbc_catalog.db.table`               | 0.9.0-incubating |
+| JDBC v1      | `spark_catalog.${schemaName}.${tableName}`  | `spark_catalog.postgres.public.table` | 0.9.0-incubating |
+
+When accessing datasets by location (e.g., `SELECT * FROM parquet.${dataset_path}`), the name is derived from the physical path:
+
+| Location Type  | Dataset name                                  | Example                               | Since Version    |
+|----------------|-----------------------------------------------|---------------------------------------|------------------|
+| GVFS location  | `${catalogName}.${schemaName}.${filesetName}` | `fileset_catalog.schema.fileset_a`    | 0.9.0-incubating |
+| Other location | location path                                 | `hdfs://127.0.0.1:9000/tmp/a/student` | 0.9.0-incubating |
+
+For GVFS location, the plugin add `fileset-location` facets which contains the location path.
+
+```json
+"fileset-location" :
+{
+"location":"${gvfs-virutal-location}",
+"_producer":"https://github.com/datastrato/...",
+"_schemaURL":"https://raw.githubusercontent...."
+}
+```
+
+## How to use 
+
+1. Download [Gravitino OpenLineage plugin jar](https://github.com/datastrato/gravitino-openlineage-plugins/tree/main/spark-plugin/) and place it to the classpath of Spark.
+2. Add configuration to the Spark to enable lineage collection.
+
+Configuration example For Spark shell:
+
+```shell
+./bin/spark-sql -v \
+--jars /${path}/openlineage-spark_2.12-${version}.jar,/${path}/gravitino-spark-connector-runtime-3.5_2.12-${version}.jar \
```

Comment:
Is this the openlineage jar, or Gravitino openlineage jar? If latter, maybe we should specify it more explicitly.

Reply:
updated